examples/nidaqmx_analog_input/_helpers.py

```diff
@@ -206,18 +206,20 @@ def verbosity_option(func: F) -> F:
         "verbosity",
         count=True,
         help="Enable verbose logging. Repeat to increase verbosity.",
-    )(func)
+    )(
+        func  # type: ignore
+    )
 
 
 def grpc_device_options(func: F) -> F:
     """Decorator for NI gRPC Device Server command line options."""
-    use_grpc_device_option = click.option(
+    use_grpc_device_option: Any = click.option(
```

Comment:
better to use a data structure if any than using `Any`

Reply:
I tried the expected type which is custom type `F`. 

Even then, I am getting the below error.

![image](https://github.com/ni/measurementlink-python/assets/13461073/3ff64286-3d75-4334-9ed6-feeb2a42f023)